controllers/submariner/gateway_resources.go

```diff
@@ -199,6 +200,7 @@ func newGatewayPodTemplate(cr *v1alpha1.Submariner, name string, podSelectorLabe
 						{Name: "SUBMARINER_HEALTHCHECKENABLED", Value: strconv.FormatBool(healthCheckEnabled)},
 						{Name: "SUBMARINER_HEALTHCHECKINTERVAL", Value: strconv.FormatUint(healthCheckInterval, 10)},
 						{Name: "SUBMARINER_HEALTHCHECKMAXPACKETLOSSCOUNT", Value: strconv.FormatUint(healthCheckMaxPacketLossCount, 10)},
+						{Name: "SUBMARINER_GWMETRICSPORT", Value: constants.MetricsGWPort},
```

Comment:
Since the env var is only consumed inside that specific container, I think we can use a simple consistent name.

```suggestion
						{Name: "SUBMARINER_METRICS_PORT", Value: constants.MetricsGWPort},
```